### DISCUSSION TITLE
Does ElevenClock support adding two clocks to the taskbar side by side?

### DISCUSSION THREAD

**Discussion by davehouser1** [martinet101/ElevenClock#1378] [General]
I want to display two clocks in the taskbar, right hand side, side by side. I want one clock to show EST -5 GMT in 12 hour time, and the other clock is UTC in 24 hour time. I do not want to hover over the clock to see these two times, I do not want to have to click it. I want them to show in the taskbar, side by side, at all times. Be aware, I do not want to use "Show a second clock on the other end of taskbar".

I have multiple monitors, however I do not care if these two clocks are on all monitors. Its fine if the clocks are just on the main monitor. 
I read through [this issue](https://github.com/marticliment/ElevenClock/issues/1079), however the last post says this:

> Due to the technical diffculty and the need of a complete rewrite of the base code, ElevenClock will not support more that two clocks per monitor.

This last post is not clear to me. "Will not support **more that two clocks** per monitor".
Does this mean that ElevenClock can support two clocks, but not more than two? Or, no, only one clock, you cant have two. I did try installing [ElevenClock 4.0.1 beta 1](https://github.com/marticliment/ElevenClock/releases/tag/4.0.1-beta1), but it was not apparent how to make multiple clocks show up. 

Does ElevenClock support two clocks, side by side, different time zones, and formats? 

If ElevenClock does support this feature, can someone direct me to instructions / documentation on how to set this up? 

**UPDATE:** Tinkering with clock apperance I was able to use this format to show what I need:
```
%#I:%M %p
(%H:%M{sec+18000}) UTC
%m/%d/%Y
```

Which creates this:
```
  10:16 AM
 15:16 UTC
11/13/2024
```

This works well. However I have a few questions. 
1. Is there an easier way to get the UTC time in the text? I see there is a `%z` and `%Z` options in the python date format [found here](https://docs.python.org/3/library/datetime.html#format-codes), but they don't seem to do anything when applying. 
2. Will this time be offset for daylight savings / fall back time changes? Will the time update when those time changes happen?
3. If I put too much format in this, like adding `%A` on a new line, or putting the 12 hour EST time next to the 24 hour UTC time, it takes up space on the taskbar and hides other taskbar objects, is there a way to have Elevelclock not cover up other icons and just shift them over?
4. Is there a way to right or center justify the text? Seems stuck at left justify.

**Comment by marticliment** [comment]
1. No, Python's %Z aren't properly explained on the docs, and they are expected to show empty here
2. No, You'd need to adjust this manually
3. No, ElevenClock can't move the icons.
4. You can find this setting under "Clock Appearance"

**Reply by davehouser1** [reply]
Thanks for the reply.
Can you comment on my original question, Does ElevenClock support two clocks, side by side, in the taskbar different time zones, and formats?

**Reply by marticliment** [reply]
You can enable a second clock from ElevenClock settings, and it will show on the other side of the taskbar.
You can also right-click a clock -> tools -> settings for this clock, and you will be able to personalize only that clock
